openmdao/docs/openmdao_book/tests/test_jupyter_output_linting.py

```diff
@@ -141,11 +148,9 @@ def test_assert(self):
                         if "allow-assert" in tags:
                             continue
 
-                    code = ''.join(block['source'])
-                    if 'assert' in code:
-
-                        msg = f"Assert found in a code block in {file}. "
-                        self.fail(msg)
+                    for line in block['source']:
```

Comment:
How much speedup in the doc build did we get from the changes?

Reply:
My motivation for making changes was to remove the os.walk from the top level scope and put it in a function so it wouldn't execute on import.  Not a huge deal with a test file I suppose, but I fixed it anyway, and while I was in there I saw a few other things that I thought could be improved.  Sometimes I get a little carried away. If you want me to revert the changes I will.  Speed wise I would expect the new version to be slightly faster but I  didn't spend the time to benchmark it.

Reply:
No, I think this is fine. I just thought you'd found a bottleneck for how slow it builds.